docs/source/rclone.rst

```diff
@@ -0,0 +1,420 @@
+       \ "publicRead"
+     5 / Project team owners get OWNER access, and all Users get WRITER access.
+       \ "publicReadWrite"
+    bucket_acl> 2
+    Location for the newly created buckets.
+    Choose a number from below, or type in your own value
+     1 / Empty for default location (US).
+       \ ""
+     2 / Multi-regional location for Asia.
+       \ "asia"
+     3 / Multi-regional location for Europe.
+       \ "eu"
+     4 / Multi-regional location for United States.
+       \ "us"
+     5 / Taiwan.
+       \ "asia-east1"
+     6 / Tokyo.
+       \ "asia-northeast1"
+     7 / Singapore.
+       \ "asia-southeast1"
+     8 / Sydney.
+       \ "australia-southeast1"
+     9 / Belgium.
+       \ "europe-west1"
+    10 / London.
+       \ "europe-west2"
+    11 / Iowa.
+       \ "us-central1"
+    12 / South Carolina.
+       \ "us-east1"
+    13 / Northern Virginia.
+       \ "us-east4"
+    14 / Oregon.
+       \ "us-west1"
+    location> 12
+    The storage class to use when storing objects in Google Cloud Storage.
+    Choose a number from below, or type in your own value
+     1 / Default
+       \ ""
+     2 / Multi-regional storage class
+       \ "MULTI_REGIONAL"
+     3 / Regional storage class
+       \ "REGIONAL"
+     4 / Nearline storage class
+       \ "NEARLINE"
+     5 / Coldline storage class
+       \ "COLDLINE"
+     6 / Durable reduced availability storage class
+       \ "DURABLE_REDUCED_AVAILABILITY"
+    storage_class> 5
+    Remote config
+    Use auto config?
+     * Say Y if not sure
+     * Say N if you are working on a remote or headless machine or Y didn't work
+    y) Yes
+    n) No
+    y/n> y
+    If your browser doesn't open automatically go to the following link: http://127.0.0.1:53682/auth
+    Log in and authorize rclone for access
+    Waiting for code...
+    Got code
+    --------------------
+    [remote_files_for_cgap_submission]
+    type = google cloud storage
+    client_id =
+    client_secret =
+    token = {"AccessToken":"xxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","RefreshToken":"x/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx_xxxxxxxxx","Expiry":"2014-07-17T20:49:14.929208288+01:00","Extra":null}
+    project_number = 12345678
+    object_acl = private
+    bucket_acl = private
+    --------------------
+    y) Yes this is OK
+    e) Edit this remote
+    d) Delete this remote
+    y/e/d> y
+
+After the Google Cloud connection is configured, you should be able to combine them
+to submit your submission files directly.
+
+.. code-block:: bash
+
+    $ rclone copy remote_files_for_cgap_submission:your-private-gcloud-bucket/fastq1.fastq.gz cgap_s3_submission:cgap-submission-bucket/case0001/fastq1.fastq.gz
+    $ rclone copy remote_files_for_cgap_submission:your-private-gcloud-bucket/fastq2.fastq.gz cgap_s3_submission:cgap-submission-bucket/case0001/fastq2.fastq.gz
+
+These transfers may take some significant time but should still be faster than if
```

Comment:
I see highlighting here of `time` and `if` in github and wonder if that's because maybe two blank lines are required to end a code block. Not sure about that. But it's odd.